model_templates/inference/r_unstructured/README.md

```diff
@@ -0,0 +1,121 @@
+## R Unstructured Inference Model Template
+
+This model is intended to work with the [R Drop-In Environment](../../../public_dropin_environments/r_lang/).
+or any other R environment as it doesn't require any specific packages.
+
+The data is just a text file [unstructured_data.txt]:(../../../tests/testdata/unstructured_data.txt).
+
+The model doesn't use any artifacts, only `custom.R` which implements the `score_unstructured` method and performs a word count on the input data.
+
+## Instructions
+Create a new custom model with these files and use it locally in a virtual environment or with the R Drop-In Environment.
+
+### To run locally using 'drum'
+File paths are relative to `./datarobot-user-models`:   
+`drum score --code-dir model_templates/inference/r_unstructured --target-type unstructured --input tests/testdata/unstructured_data.txt`
+
+#### Running with additional params
+The following example shows how to control the types of the values passed into the `score_unstructured` hook.
+
+Consider the following signature: `score_unstructured <- function(model, data, query, ...)`
```

Comment:
I think it's worth describing in more detail what a query input should look like (from a drum score --query perspective), and how it should be parsed in R. Although you have the example below, it'd be nicer to have some documentation rather than relying on people to read the implementation in `custom.R` directly.

Reply:
Query should not be documented so far. Rabbit hole of feature we wont yet use.

Reply:
We have query param in `drum score --help` ?
Let's keep as for today, and go over all the docs in a week or so. There are a couple of things that has to be changed.